### ISSUE TITLE
Move all li elements to ul element

### ISSUE THREAD

**Issue by grzanka** [yaptide/ui#1415]
All li elements should be inside ul element ((nav-items like editor, input files etc.)

**Comment by grzanka** [comment]
![image](https://github.com/yaptide/ui/assets/7374733/be174e8e-2e79-4d58-9583-28536177bb63)
